applications/blink_raw/targets/freertos.mips4k.pic32mx.lccbuffer/pic32mx_cf_init.c

```diff
@@ -0,0 +1 @@
+../../../../boards/mc-pic32mx-lccbuffer/pic32mx_cfg_init.c
```

Comment:
just a cosmetic comment: the name of this file is mistyped (missing a 'g')

note: if you change the file name you may have to do a clean first before the next build.